### ISSUE TITLE
Job submission form: remember folder location for read 2

### ISSUE THREAD

**Issue by mshukla1** [PATRIC3/patric3_website#1982]
While selecting paired end read files, I browse through the directory structure, go to the right folder, and then select read file 1. Then, I need to again go through the same directory structure to select second file!! 

UI should remember the location of the first file selected and directly show the same folder while selecting file 2. 

We do it for subsequent file selections, so we should be able to do it for read file 2 as well. 

-Maulik

<img width="374" alt="screen shot 2018-05-08 at 2 00 36 pm" src="https://user-images.githubusercontent.com/1236463/39777307-d332c840-52c8-11e8-85c4-90fb679585c2.png">

**Comment by dawenx** [comment]
While I agree that it would be convenient for the user, I personally don't like it as it somewhat is a hack: we are using WorkspaceObjectSelector here and the default behavior is to go to user's home, now we want to change this default behavior for READ2 in paired read library selection.

What if a user chooses a READ2 first? Do we then make READ1 go to READ2's directory as well? What if READ1 is selected, then unselected, should we keep the default folder of READ2 alway the same as READ1?

@hyoo do you have suggestion? I can work on it once we agree what needs to be done.

**Comment by nconrad** [comment]
I agree this should be added.  I had removed the whole concept of remembering folders because the workspace browser (not the object selector) was also using that state.  However, for paired-end read selection, this would be *extremely* useful and speed things up greatly in workshops.  

This is likely related to #1306.

I think if read2 is selected first, read1 should have the same default folder.   If read1 is selected and then unselected, I think it is fine to have the default for read 1/2 be the previously selected folder.  The potential confusion there is rare occurrence.

**Comment by nconrad** [comment]
Worst case use case would be if users have several paired-end reads located in different folders or such.  In that case, I think they can learn to go up a directory or such.  Adding the above would still be an improvement.

**Comment by dawenx** [comment]
@nconrad thanks for your input and that makes sense. i will work on it then.

**Comment by mshukla1** [comment]
Thanks for raising the questions, Dawen. For good design, we do need to think about possible scenarios and how to handle them, before we implement. 

I leave the details to you, Neal and harry. I think we should try it first on with one service, test it and then apply to other services.
